jck/jck.mk

```diff
@@ -61,6 +61,11 @@ OTHER_OPTS=
 # if JDK_IMPL is openj9 or ibm
 ifneq ($(filter openj9 ibm, $(JDK_IMPL)),)
  OTHER_OPTS=-Xtrace:maximal=all{level2}
+ # if JDK is 8
+ if (JDK_VERSION != "8") {
+	OTHER_OPTS += " --enable-preview"
+	}
+ endif
```

Comment:
This will not work in make. Please check make syntax. https://www.gnu.org/software/make/manual/html_node/Conditional-Syntax.html

Reply:
Hi @llxia, thanks for the review and the link! Does it work now?

Reply:
The logic is not correct. Please check the original logic as reference: https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/buildenv/jenkins/testJobTemplate#L165

- `-Xfuture` is added only if it is openj9/ibm
- `--enable-preview` is added if JDK_VERSION != "8". The comment is wrong and we do not need filter, we can just use a simple `ifneq`
- `-Dcom.ibm.tools.attach.enable=yes` needs to be added for zos